tools/sgen/input/FluentUIStyle.yml

```diff
@@ -296,3 +302,10 @@ AP_MSFDrawerTokens:
   shadow2Blur: $Shadow.shadow28.blur2
   shadow2OffsetX: $Shadow.shadow28.x2
   shadow2OffsetY: $Shadow.shadow28.y2
+  minMargin: [horizontal: $Spacing.xxxlarge, vertical: $Spacing.large]
+  cornerRadius: $Border.radius.xxlarge
+  dropShadowRadius: $Border.radius.medium
```

Comment:
The tokens here are not being used anywhere. Is that right?
- dropShadowRadius
- dropShadowOffset
- dropShadowOpacity

If so, they should be removed from here.